### PR TITLE
Change flexbox to grid

### PR DESCRIPTION
**Pull request by blacksam07**
## Summary
- Change Flexbox to Grid

## Trello card
[12-rediseño-de-la-página](https://trello.com/c/IuZxSLH5/12-redise%C3%B1o-de-la-p%C3%A1gina)

## Screenshots
- Table grid
table-grid
- Mobile grid
modile-grid

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -118,35 +139,49 @@ $max-width-mobile: 1024px;
...
+    margin: auto;
   
     &::after {
-      bottom: 240px;
-      height: calc($badge-size * 30%);
+      bottom: 250px;
+      height: calc(100px * 0.7);
       left: 165px;
-      width: calc($badge-size * 30%);
+      width: calc(100px * 0.7);
```
Comment (betoobernal) [inline scss/detail.scss]: well yes, but actually no.jpg 
In the function `calc`, this is equal always to 70px. You're going to look its magic when you use it with relative units

File: scss/detail.scss [hunk 2]
```diff
@@ -3,24 +3,30 @@ $max-width-mobile: 1024px;
 
 .base-span {
   color: $gray;
-  font-size: 20px;
-  margin-left: 10px;
+  font-size: 24px;
+  padding-left: 10px;
```
Comment (CrisVega08) [inline scss/detail.scss]: sort

File: scss/detail.scss [hunk 3]
```diff
@@ -31,20 +37,30 @@ $max-width-mobile: 1024px;
   margin: auto;
   padding: 80px;
   width: fit-content;
+  display: grid;
```
Comment (CrisVega08) [inline scss/detail.scss:37]: sort

File: scss/detail.scss [hunk 4]
```diff
@@ -31,20 +37,30 @@ $max-width-mobile: 1024px;
...
   box-shadow: 4px 4px 10px 0 $black-lighter-5;
-  display: flex;
-  height: 415px;
-  margin-top: 50px;
-  width: 900px;
+  display: grid;
+  grid-area: content;
+  grid-template-columns: 1fr 4fr;
+  grid-template-rows: 1fr 1fr 8fr;
+  grid-template-areas: "space1 title" "space2 subtitle" "image-cover description";
```
Comment (CrisVega08) [inline scss/detail.scss]: use more descriptive name

File: scss/detail.scss [hunk 5]
```diff
@@ -56,29 +72,28 @@ $max-width-mobile: 1024px;
...
 
 .container-item {
-  margin: 25px;
+  margin: 15px 30px;
 }
 
 .container-data {
-  flex-grow: 2;
+  grid-area: description;
+  height: 65%;
```
Comment (CrisVega08) [inline scss/detail.scss:80]: I am no sure for this value, is relative ??

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/detail.scss [hunk 6]
```diff
@@ -118,35 +139,49 @@ $max-width-mobile: 1024px;
...
+    width: auto;
 
-  .bold-text,
-  .base-span {
-    font-size: 14px;
+    &::after {
+      background-color: $earls-green;
+      content: '';
+      height: 4px;
+      left: -120px;
```
Comment (betoobernal) [inline scss/detail.scss]: use positive values.

File: scss/detail.scss [hunk 7]
```diff
@@ -56,29 +72,28 @@ $max-width-mobile: 1024px;
...
 .container-data {
-  flex-grow: 2;
+  grid-area: description;
+  height: 65%;
+  margin: auto;
+  width: 100%;
 }
 
 .book-cover {
   height: 368px;
```
Comment (betoobernal) [inline scss/detail.scss]: use multiples of 5

File: scss/detail.scss (hunk 5 above)
Comment (blacksam07) [reply scss/detail.scss:80]: yes, is relative

**Review by CrisVega08 — APPROVED** [review]

**Review by betoobernal — APPROVED** [review]

**Pull request merged**